pkg/tf2pulumi/convert/testdata/same-id-output-no-rewrite/main.tf

```diff
@@ -0,0 +1,12 @@
+#if EXPERIMENTAL
```

Comment:
Does this have to be just for EXPERIMENTAL, doesn't it work for the old converter as well?

Reply:
The old converter does something weird with the order of node declaration so it chooses to mangle the config variables first and leaves outputs as is